### PR TITLE
Add cli cmds for dumping bookings from a given date to json and do followup bookings

### PR DESCRIPTION
**Pull request by calbrecht**
Usage:  

for a whole day

`hug -f main.py -c get_bookings_created_at 2021-02-05 | hug -f main.py -c batch_book_followup`

or a single booking

`hug -f main.py -c get_bookings_created_at 2021-02-05T14:23:43.987654 | hug -f main.py -c batch_book_followup`

Although this examples are stupid, which could be observed if reading the code, they demonstrate the fact that the command `batch_book_followup` expects the output of `get_bookings_created_at` as input on `stdin`

### CODE REVIEW

File: termine-be/cli.py [hunk 1]
```diff
@@ -296,3 +297,124 @@ def load_frontend_config(db: directives.PeeweeSession, frontend_config_file: hug
...
+    """
+    return Booking.select(Booking).where(Booking.booked_by == user).count() > 0
+
+
+@hug.cli(output=hug.output_format.pretty_json)
+def book_followup(db: directives.PeeweeSession, booking: hug.types.json):
+    """
+    args: BOOKING_JSON
+    """
+    if has_booked_appointment(db, booking["booked_by"]):
```
Comment (calbrecht) [inline termine-be/cli.py:375]: This is probably wrong. At least after this command was ran once, there exists a booking made by that user and all following bookings will be discarded.

File: termine-be/cli.py [hunk 2]
```diff
@@ -296,3 +297,124 @@ def load_frontend_config(db: directives.PeeweeSession, frontend_config_file: hug
...
+        print(f"User {booking['booked_by']} already booked at least one appointment.")
+        #return None
```
Comment (calbrecht) [inline termine-be/cli.py:377]: Uh, oh!

**Comment by calbrecht** [comment]
Another point worth mentioning is, the followup booking will get another `secret` that the person who will have the appointment will not have knowledge.

File: termine-be/cli.py [hunk 3]
```diff
@@ -296,3 +297,147 @@ def load_frontend_config(db: directives.PeeweeSession, frontend_config_file: hug
...
+def book_followup(db: directives.PeeweeSession, booking: hug.types.json):
+    """
+    args: BOOKING_JSON
+    """
+    #if has_booked_by(db, booking["booked_by"]):
+    #    print(f"User {booking['booked_by']} already booked at least one appointment.")
+    #    return None
+    if has_booking(db, booking):
+        print(f"A booking for person from {booking} already exists.")
+        return None
```
Comment (vschaeidt) [inline termine-be/cli.py:400]:
```suggestion
    if has_booked_by(db, booking["booked_by"]):
        print(f"User {booking['booked_by']} already booked at least one appointment.")
        return None
    #if has_booking(db, booking):
    #    print(f"A booking for person from {booking} already exists.")
    #    return None
```

**Pull request merged**